hugegraph-llm/src/hugegraph_llm/config/models/base_prompt_config.py

```
@@ -60,67 +68,73 @@
-            file.write(yaml_content)
+            yaml.dump(data, file, allow_unicode=True, sort_keys=False, default_flow_style=False)
 
     def generate_yaml_file(self):
         if os.path.exists(yaml_file_path):
             log.info("%s already exists, do you want to override with the default configuration? (y/n)", yaml_file_path)
             update = input()
             if update.lower() != "y":
                 return
-            self.save_to_yaml()
+
+        if self.llm_settings.language.lower() == "cn":
```

Pylint (3.3.7)

[error] Instance of 'BasePromptConfig' has no 'llm_settings' member

(E1101)